scripts/Test-Signatures.ps1

```diff
@@ -0,0 +1,209 @@
+[CmdletBinding()]
+param (
+    [Parameter()]

```

Comment:
Nit: this is not needed when you don't customize anything, did you mean to use `Mandatory`?

Reply:
No, I just used the snippet :)